lib/components/src/blocks/Preview.tsx

```diff
@@ -132,6 +156,23 @@ const PositionedToolbar = styled(Toolbar)({
   height: 40,
 });
 
+const Relative = styled.div({
+  overflow: 'hidden',
+  position: 'relative',
+});
+
+const getLayout = (children: ReactElement[]) => {
```

Comment:
AFAICT this uses the first story layout if there are multiple stories with different layouts, or `padded` if there are non-story children. WDYT about using the story layout if all children are stories and their layouts match, and `padded` layout in any other case. Seems a little simpler and more predictable?